gems/aws-sdk-core/spec/aws/instance_profile_credentials_spec.rb

```diff
@@ -422,39 +404,38 @@ module Aws
         expect(creds.account_id).to be_nil
       end
 
-      it 'Test IMDS credentials provider with a given profile name when account ID is unavailable when profile is invalid throws an error' do
-        stub_request(:get, "#{ipv4_endpoint}#{extended_path}my-profile-0008").to_return(status: 404)
-        stub_request(:get, "#{ipv4_endpoint}#{fallback_path}my-profile-0008").to_return(status: 404)
+      it 'Test IMDS credentials provider with a given profile name when account ID is unavailable'\
```

Comment:
I know rubocop complains about this being "too long" but I think it's fine to not truncate this - it actually makes searching the file for this test name difficult.